clientv3/watch.go

```diff
@@ -28,6 +28,7 @@ import (
 const (
 	EventTypeDelete = mvccpb.DELETE
 	EventTypePut    = mvccpb.PUT
+	EventTypeExpire = mvccpb.EXPIRE
```

Comment:
Uh-oh-- there's not? If a key expires, and there's a watch on it, and we don't get an event, I would consider this a major bug :(